app/views/candidate_interface/personal_statement/_form.html.erb

```diff
@@ -13,7 +13,10 @@
   <li>any past experience working with children or young people, and what you learnt</li>
   <li>your thoughts on welfare and education</li>
 </ul>
-<p class="govuk-body">If you need help, you can speak to a <%= govuk_link_to t('service_name.get_into_teaching'), t('get_into_teaching.url_get_an_advisor') %> adviser. They’re all experienced teachers who can give you advice on applying for teacher training.</p>
+
+<%= govuk_inset_text do %>
+  <p class="govuk-body"><%= govuk_link_to 'Find out if you’re eligible for a teacher training adviser', t('get_into_teaching.url_get_an_advisor') %> to get advice on your application from an experienced teacher.</p>
+<% end %>
```

Comment:
@EmmaFrith can we drop the inset text here, and just have it as a plain paragraph? I've increasingly found that the inset text design makes people less likely to read the text rather than more... 🤦